$NetBSD: patch-make_autoconf_buildjdk-spec.gmk.in,v 1.2 2026/09/06 16:53:12 tnn Exp $

Use the same variant for the buildjdk as the target jdk.
This is needed for native builds on platforms where hotspot is broken or not
supported. In that case the buildjdk can't be hardcoded to "server" but must
use "zero". This is fine for pkgsrc since we only ever build one variant.

--- make/autoconf/buildjdk-spec.gmk.in.orig	2026-07-23 23:33:34.000000000 +0000
+++ make/autoconf/buildjdk-spec.gmk.in
@@ -94,8 +94,9 @@ ENABLE_DEBUG_SYMBOLS := false
 # Save speed and disk space by not enabling debug symbols for the buildjdk
 ENABLE_DEBUG_SYMBOLS := false
 
-JVM_VARIANTS := server
-JVM_VARIANT_MAIN := server
+JVM_VARIANTS :=  @JVM_VARIANTS@
+JVM_VARIANT_MAIN := @JVM_VARIANT_MAIN@
+JVM_FEATURES_zero := cds g1gc serialgc zero
 JVM_FEATURES_server := cds compiler1 compiler2 g1gc serialgc
 
 # Some users still set EXTRA_*FLAGS on the make command line. Must
